docs/Documentation/run_RBM.md

```diff
@@ -89,9 +87,9 @@ This will create the executable,
 
 For this example, copy **rout** to the folder **Test_Case**
 
-Create control file, **salmon.inp_DA**, to run the modified Lohmann routing model using the file, **Rout.Cells.init**, the first time and the file, **Rout.Cells**, if the routing model, **rout** is run again for the same set of unit hydrographs. The input file (in this example, **salmon.inp_DA**) is similar to that described on the VIC model Web site with the exception of the addition the Leopold parameters. The routing model requires a file with unit hydrographs for each cell, or, in this case, a file, **UH_ALL**, with a single unit hydrograph that is applied to all grid cells.
+Create the control file **salmon.inp_DA** (see [Figure 3](../figures/tutorial_f3.png) for an example) to run the modified Lohmann routing model using the file **Rout.Cells.init** (for the first time) or the file **Rout.Cells** if the routing model **rout** is run again for the same set of unit hydrographs. The input file (in this example, **salmon.inp_DA**) is similar to that described on the VIC model Web site with the exception of the addition the Leopold parameters. The routing model requires a file with unit hydrographs for each cell, or, in this case, a file, **UH_ALL**, with a single unit hydrograph that is applied to all grid cells.
 
-### Step 6 Run rounting model
+### Step 6 Run routing model
```

Comment:
- At line 96 in step 6 - "to create the direct RBM input files of flow and heat budget" - also extra comma here
- Give link to Appendix A (same for all the places where Appendix A is referred)
- I don't think the variables in the flow and heat files are all "state variables". Just "variables"